Description: Adding Copyright

---
The information above should follow the Patch Tagging Guidelines, please
checkout http://dep.debian.net/deps/dep3/ to learn about the format. Here
are templates for supplementary fields that you might want to add:

Origin: <vendor|upstream|other>, <url of original patch>
Bug: <url in upstream bugtracker>
Bug-Debian: https://bugs.debian.org/<bugnumber>
Bug-Ubuntu: https://launchpad.net/bugs/<bugnumber>
Forwarded: <no|not-needed|url proving that it has been forwarded>
Reviewed-By: <name and email of someone who approved the patch>
Last-Update: <YYYY-MM-DD>

--- fritzident-0.1.orig/debug.c
+++ fritzident-0.1/debug.c
@@ -1,7 +1,7 @@
 /*
  * debug.c
  *
- * Copyright (C) 2013 - Unknown
+ * Copyright (C) 2013 - Andre Larbiere <andre@larbiere.eu>
  *
  * This program is free software; you can redistribute it and/or modify
  * it under the terms of the GNU General Public License as published by
--- fritzident-0.1.orig/netinfo.c
+++ fritzident-0.1/netinfo.c
@@ -1,7 +1,7 @@
 /*
  * netinfo.c
  *
- * Copyright (C) 2013 - Unknown
+ * Copyright (C) 2013 - Andre Larbiere <andre@larbiere.eu>
  *
  * This program is free software; you can redistribute it and/or modify
  * it under the terms of the GNU General Public License as published by
--- fritzident-0.1.orig/netinfo.h
+++ fritzident-0.1/netinfo.h
@@ -1,7 +1,7 @@
 /*
  * netinfo.h
  *
- * Copyright (C) 2013 - Unknown
+ * Copyright (C) 2013 - Andre Larbiere <andre@larbiere.eu>
  *
  * This program is free software; you can redistribute it and/or modify
  * it under the terms of the GNU General Public License as published by
--- fritzident-0.1.orig/userinfo.c
+++ fritzident-0.1/userinfo.c
@@ -1,7 +1,7 @@
 /*
  * userinfo.c
  *
- * Copyright (C) 2013 - Unknown
+ * Copyright (C) 2013 - Andre Larbiere <andre@larbiere.eu>
  *
  * This program is free software; you can redistribute it and/or modify
  * it under the terms of the GNU General Public License as published by
--- fritzident-0.1.orig/userinfo.h
+++ fritzident-0.1/userinfo.h
@@ -1,7 +1,7 @@
 /*
  * userinfo.h
  *
- * Copyright (C) 2013 - Unknown
+ * Copyright (C) 2013 - Andre Larbiere <andre@larbiere.eu>
  *
  * This program is free software; you can redistribute it and/or modify
  * it under the terms of the GNU General Public License as published by
